tests/data_files/Makefile

```diff
@@ -169,6 +199,22 @@ server7_int-ca-exp.crt: server7.crt test-int-ca-exp.crt
 	cat server7.crt test-int-ca-exp.crt > $@
 all_final += server7_int-ca-exp.crt
 
+cli2.crt.der: cli2.crt
+	$(OPENSSL) x509 -in $< -out $@ -inform PEM -outform DER
+all_final += cli2.crt.der
+
+ cli2.key.der: cli2.key
```

Comment:
There are heading spaces in this recipe and the two following ones.

Reply:
Thanks! Fixed in b9eee1.